lib/generator/files/generator_cases.rb

```diff
@@ -4,7 +4,7 @@ module GeneratorCases
       module_function
 
       def available(track_path)
-        generator_glob = File.join(track_path, "{lib,exercises}", '**', '*_cases.rb')
+        generator_glob = File.join(track_path, "exercises", '**', '*_cases.rb')
```

Comment:
This probably maybe could possibly refer to the full correct path and only glob the `*_cases.rb`